base/db/DESCRIPTION

```diff
@@ -3,16 +3,41 @@ Type: Package
 Title: PEcAn functions used for ecological forecasts and reanalysis
 Version: 1.6.0
 Date: 2018-08-20
-Authors@R: c(person("Mike","Dietze"),
-             person("David","LeBauer"),
-             person("Xiaohui", "Feng"),
-             person("Dan"," Wang"),
-             person("Carl", "Davidson"),
-             person("Rob","Kooper"),
-             person("Shawn", "Serbin"))
-Author: David LeBauer, Mike Dietze, Xiaohui Feng, Dan Wang, Carl
-    Davidson, Rob Kooper, Shawn Serbin
-Maintainer: David LeBauer <dlebauer@illinois.edu>
+Authors@R: c(person("Mike", "Dietze", role = c("aut", "cre"),
```

Comment:
Check me on this, but my reading of `?person` is that only one "cre" role is allowed per package. I'm not sure why they called it "creator" and not "maintainer", but think "person who creates the releases / CRAN submissions" more than "person who wrote the first line of code"

Reply:
Yeah you were right.

Reply:
And just to confirm: The change in maintainer from @dlebauer to @mdietze is intentional? If so, this looks ready to merge.

Reply:
Not intentional on my side! I'd prefer that @dlebauer remain in charge of the DB package